Source/Core/WinUpdater/WinUpdater.rc

```diff
@@ -0,0 +1,71 @@
+// Microsoft Visual C++ generated resource script.
+//
+#include "resource.h"
+
+#define APSTUDIO_READONLY_SYMBOLS
```

Comment:
We considerably reduced the contents of `Source/Core/DolphinQt/DolphinQt.rc` because the majority of it has no use anyways (as we don't really use the IDE provided designers and stuff). Thoughts on doing the same here?

Reply:
I thought the auto-generated files would be fine as they are. But I'll remove the unused content in `WinUpdater.rc` then. Thanks for pointing it out.